DER/FxDerivatives/FxOptions.rdf

```diff
@@ -64,193 +68,109 @@
-		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeDerivativeContract"/>
+	<owl:Class rdf:about="&fibo-der-fx-opt;CurrencyOption">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Option"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-fx-opt;buyer"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionBuyer"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-fx-opt;hasSeller"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionSeller"/>
+				<owl:onProperty rdf:resource="&fibo-der-fx-opt;hasBuyingCurrency"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;Currency"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;embodies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-opt;ForeignExchangeOptionTransaction"/>
+				<owl:onProperty rdf:resource="&fibo-der-fx-opt;hasSellingCurrency"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;Currency"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">foreign exchange (FX) option</rdfs:label>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-der-fx-opt;ForeignExchangeOptionBuyer">
-		<rdfs:label xml:lang="en">fx option buyer</rdfs:label>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-der-fx-opt;ForeignExchangeOptionLeg">
-		<rdfs:label xml:lang="en">fx option leg</rdfs:label>
-		<skos:definition xml:lang="en">A standard FX OTC option (European or American) which may be a complete trade in its own right or part of a trade strategy. FpML A type that is used for describing a standard FX OTC option (European or American) which may be a complete trade in its own right or part of a trade strategy.</skos:definition>
-	</owl:Class>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-fx-opt;ForeignExchangeOptionPremium">
-		<rdfs:label xml:lang="en">fx option premium</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-fx-opt;ForeignExchangeOption"/>
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-		<skos:definition xml:lang="en">FpML &apos;Premium amount or premium installment amount for an option.&apos;</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:Class rdf:about="&fibo-der-fx-opt;ForeignExchangeOptionSeller">
-		<rdfs:label xml:lang="en">fx option seller</rdfs:label>
+		<rdfs:label xml:lang="en">currency option</rdfs:label>
+		<skos:definition xml:lang="en">A contract that grants the holder the right, but not the obligation, to buy or sell currency at a specified exchange rate during a specified period of time.</skos:definition>
```

Comment:
important - need to say that it's a specific amount of the currency. Need a restriction for the amount. Is the contract in terms of the buying or the selling currency e.g. do you say I want to buy $100 of GBP or 100 GBP?